src/Symfony/Component/Console/Helper/ProgressIndicator.php

```diff
@@ -129,8 +129,6 @@ public function advance()
 
     /**
      * Finish the indicator with message.
-     *
-     * @param $message
```

Comment:
Useless

Reply:
I think that's the only change we might wanna keep. See @wouterj's comment.
Also 5.4 is for bug fixes only, and this is not one.

Reply:
Done